.gitignore

```diff
@@ -360,4 +360,7 @@ MigrationBackup/
 .ionide/
 
 # Fody - auto-generated XML schema
-FodyWeavers.xsd
\ No newline at end of file
+FodyWeavers.xsd
+
```

Comment:
Was this change intentional? If so, could you provide more information about it? I couldn't find any details online regarding the change.

Reply:
I don't know why "FodyWeavers.xsd" was modified, I only added the JetBrains IDE section.
.idea is where JetBrains IDEs like Rider store IDE specific settings and stuff.